views/js/qtiCreator/tpl/forms/interactions/media.tpl

```diff
@@ -8,11 +7,12 @@
         </label>
     </div>
 
-    <h3 class="media-sizer-panel-label">{{__ 'Size and position'}}</h3>
-    <div class="panel media-sizer-panel">
-        <!-- mediaEditorComponent goes here -->
-    </div>
-
+    {{#unless isAudio}}
+        <h3 class="media-sizer-panel-label">{{__ 'Size and position'}}</h3>
+        <div class="panel media-sizer-panel">
+            <!-- mediaEditorComponent goes here -->
```

Comment:
1. Create new interaction
2. Upload _video_
3. Check 'Autostart' ---> Size slider dissapears and doesn't reappear when you uncheck it

(I'm testing this branch: https://github.com/oat-sa/extension-tao-itemqti/pull/2382)

Reply:
Fix is in progress...